go.mod

```diff
@@ -54,14 +54,14 @@ require (
 	github.com/ti-mo/conntrack v0.4.0
 	github.com/vishvananda/netlink v1.1.1-0.20211101163509-b10eb8fe5cf6
 	github.com/vmware/go-ipfix v0.5.12
-	golang.org/x/crypto v0.0.0-20220214200702-86341886e292
+	golang.org/x/crypto v0.0.0-20220427172511-eb4f295cb31f
 	golang.org/x/exp v0.0.0-20200224162631-6cc2880d07d6
-	golang.org/x/mod v0.6.0-dev.0.20220106191415-9b9b3d81d5e3
-	golang.org/x/net v0.0.0-20220722155237-a158d28d115b
-	golang.org/x/sync v0.0.0-20220601150217-0de741cfad7f
+	golang.org/x/mod v0.6.0-dev.0.20220419223038-86c51ed26bb4
+	golang.org/x/net v0.0.0-20220906165146-f3363e06e74c
+	golang.org/x/sync v0.0.0-20220722155255-886fb9371eb4
 	golang.org/x/sys v0.0.0-20220908164124-27713097b956
 	golang.org/x/time v0.0.0-20220210224613-90d013bbcef8
-	golang.org/x/tools v0.1.10-0.20220218145154-897bd77cd717
+	golang.org/x/tools v0.1.12
```

Comment:
Just to confirm: are these versions are selected according to CVEs?

Reply:
No, it is not CVE related, this is changed, when I run make tidy

Reply:
This may be confusing that the updated librarie are affected by CVEs given the title.
The libraries are not updated when I run `make tidy` locally, so I guess it's related to your env or golang version. Anyway this change shouldn't be included in a patch that fixes CVEs to avoid confusion, especially we may need to backport the patch.

Reply:
maybe it is related to the packages that are bumped by fixing CVEs

Reply:
Sorry, I'm confused. I asked if these version changes are related to CVEs you said no. What changes are made for CVEs?

Reply:
This package is not related to CVE, I mean I bumped packages for CVEs, such as `golang.org/x/net`, and then I used make tidy, go changes version for tools. So I guess maybe this change for `golang.org/x/tools`  is because I bumped some other packages. My golang version is `1.19.2`

Reply:
Got it, thanks for explanation